src/app/components/articles/article/title/title.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { Article } from 'src/app/article';
+
+@Component({
+  selector: 'app-article-title',
+  templateUrl: './title.component.html',
+  styleUrls: ['./title.component.scss']
+})
+export class TitleComponent {
+  article: Article = {
+    id: 1,
+    title: "Le caramel qui colle aux dents : pour ou contre ?",
```

Comment:
add slug entry

Reply:
Slug will be intergrated will using the datas from US-Interfaces